tests/CMakeLists.txt

```diff
@@ -1,53 +0,0 @@
-#
```

Comment:
This delete file might be why the CI build fails.

Reply:
Yes, that is one of the reasons. I put it back in the tests directory, along with other changes.